package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "private": true,
+  "scripts": {
+    "prepare": "lerna run prepare",
+    "rebuild:browser": "theia rebuild:browser",
+    "rebuild:electron": "theia rebuild:electron"
+  },
+  "devDependencies": {
+    "lerna": "2.4.0"
+  },
+  "workspaces": [
+    "github-extension",
+    "browser-app",
+    "electron-app"
+  ],
+  "dependencies": {
+    "github": "^13.1.0"
```

Comment:
we may want to omit `^` so we stick to a specific version

Reply:
fixed